invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/deposit.html

```diff
@@ -41,6 +41,11 @@
            value='{{ files | tojson }}'>
   {%- endif %}
 
+  {%- if config %}
+    <input type="hidden" name="deposits-groups-enabled"
+           value='{{ config.USERS_RESOURCES_GROUPS_ENABLED | tojson }}'>
+  {%- endif %}
+
```

Comment:
Config is always there, and I would also change the name:
```suggestion
  <input type="hidden" name="config-groups-enabled"
           value='{{ config.USERS_RESOURCES_GROUPS_ENABLED | tojson }}'>

```